komodo/yaml_file_type.py

```diff
@@ -12,24 +52,80 @@ def __init__(self, *args, **kwargs):
 
     def __call__(self, value):
         file_handle = super().__call__(value)
-        yaml = yml.safe_load(file_handle)
+        yaml = YAML()
+        yml = yaml.load(file_handle)
```

Comment:
The variable names look random here since `yaml` and `yml` have very different type. 

Maybe write it as one-line:  `yml  = YAML().load(file_handle)`